packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -344,6 +355,7 @@ export default class Reachability {
         // Close any open peerConnections
         if (peerConnectionProxy.connectionState !== CLOSED) {
           this.setLatencyAndClose(peerConnectionProxy, null);
+          this.setClientMediaIPs(peerConnectionProxy, null);
```

Comment:
better to call this before `this.setLatencyAndClose()` - see my other comment why, or maybe we don't even need to call this at all here

Reply:
switched the order.